skills/lark-doc/references/lark-doc-fetch.md

```
@@ -6,110 +6,129 @@
-
-## 重要：图片、文件、画板的处理
-
-**文档中的图片、文件、画板需要通过独立的 media shortcut 单独获取。**
+# 按关键词定位（多关键词用 | 分隔，任一命中即返回）
+lark-cli docs +fetch --api-version v2 --doc Z1Fj...tnAc \
+  --scope keyword --keyword "部署|发布|上线"
+```
 
-### 识别格式
+## 选 `--detail`（每块详细度）
+| 意图 | `--detail` | 说明 |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)